packages/render-html/src/__tests__/regression.xxx.profiler-warnings.test.ts

```diff
@@ -0,0 +1,15 @@
+import { renderHook } from '@testing-library/react-hooks';
+import useProfiler from '../hooks/useProfiler';
+
+jest.useFakeTimers();
```

Comment:
The behavior of `jest.useFakeTimers()` might change depending on the version of Jest and the config we're using here. Alternatively, I could change this to:

```ts
global.performance = {
  now: jest.fn().mockReturnValue(0)
}
```